FIAT/enriched.py

```diff
@@ -66,24 +67,37 @@ def __init__(self, A, B):
         # the entities of the constituent elements
         Adofs = A.entity_dofs()
         Bdofs = B.entity_dofs()
+
         offset = A.space_dimension()  # number of entities belonging to A
-        entity_ids = {}
+        self.entity_ids = {}
```

Comment:
It seems that the dual saves the entity ids anyway, so why do you need to store them on the element as well?